src/libpriv/rpmostree-util.c

```diff
@@ -1210,6 +1212,7 @@ rpmostree_relative_path_is_ostree_compliant (const char *path)
   g_assert (*path != '/');
   return (g_str_equal (path, "usr")   || (g_str_has_prefix (path, "usr/")
                                           && !g_str_has_prefix (path, "usr/local/")) ||
+          g_str_equal (path, "opt")   || g_str_has_prefix (path, "opt/")  ||
```

Comment:
Hmm, though that changes the meaning of this function enough that the name is no longer appropriate. :) Thinking more on this, maybe let's put this directly in the importer instead since that's where the special-handling for `opt/` lives?